drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/lir/OCLLIRStmt.java

```diff
@@ -55,6 +55,37 @@ public final void emitCode(CompilationResultBuilder crb) {
 
     }
 
+    @Opcode("BEGIN_SKIP_SEQUENCE")
+    public static class BeginSkipStmt extends AbstractInstruction {
```

Comment:
The semantics is to Relocate an instruction from within the loop to outside the loop. Therefore, let's update the name to `MarkRelocatebleInstruction`